scripts/gen-deps.sh

```diff
@@ -16,4 +16,4 @@ echo generating dependencies for tests workspace
 rm -fr tests/3rdparty
 external-tools/bazel-deps.sh generate -r "$(pwd)/tests" -s 3rdparty/maven.bzl -d dependencies.yaml
 
-./format.sh
+./scripts/format.sh
```

Comment:
I prefer to have separate PR for all these small fixes, same for gen docs. It's easier to track what has been changed later on.

Reply:
Done!